backend/editor.go

```diff
@@ -162,9 +162,11 @@ func getClipboard() (string, error) {
 }
 
 func (e *Editor) Init() {
+	log.Info("Initializing")
```

Comment:
As our logs are getting pretty noisy, it might help to make this a bit more descriptive. Maybe `Initializing Editor`?